pkg/p2p/wire/message/agreement_in_test.go

```diff
@@ -0,0 +1,84 @@
+package message
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/dusk-network/dusk-blockchain/pkg/core/consensus/key"
+	"github.com/dusk-network/dusk-blockchain/pkg/util/nativeutils/sortedset"
+	"github.com/dusk-network/dusk-crypto/bls"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAgreementCopy(t *testing.T) {
+
```

Comment:
Can probably remove this test, since we seem to test the same thing in `agreement_test.go`

Reply:
yeah, it was a leftover